app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (150, 38, [33, 4]),
+        (-5, -10, [0, 0]),
```

Comment:
The test case with negative ages might not be valid depending on the function's expected behavior. Ensure that the function raises the correct exception for invalid input types, as mentioned in the checklist.